### DISCUSSION TITLE
Remote profile avatars/headers refresh

### DISCUSSION THREAD

**Discussion by RegalMonkey** [mastodon/mastodon#17827] [Q&A] [answered]
Hello fediverse dwellers, I have a question that should be something possible with the DB/CLI but I'm not clear how.

For my instance I set up a MinIO server to handle media, and that VM then stores the media on a NAS. Some while ago when moving drives around on the NAS a good chunk of the data got lost (trying to rsync millions of tiny files made it unhappy) so I've tried to do media refreshes and for the most part have been successful.

The problem area is remote profiles, the avatars/headers don't look to be included in the pull. So I end up with a bunch of profiles that show blank boxes because the DB references MinIO and then MinIO doesn't have a related object.

Example: https://cdn.socdojo.com/mastodon/cache/accounts/avatars/000/000/399/original/9bc7094d1a3eabbb.jpeg

Is there something I can do via the CLI or DB directly to have the instance purge these references and re-pull them?

**Comment by RegalMonkey** [comment, marked as answer]
tootctl accounts refresh --all --verbose

With or without flags, modify as needed. Tip: don't go thinking that user avatars and wallpapers fall under media like me and start trying weird purging efforts to fix something simple... c'est la vie

**Reply by snow-of-the-mountain** [reply]
> tootctl accounts refresh --all --verbose
> 
> With or without flags, modify as needed. Tip: don't go thinking that user avatars and wallpapers fall under media like me and start trying weird purging efforts to fix something simple... c'est la vie

Does that also do media that those profiles uploaded as well?
